docs/installation.rst

```diff
@@ -54,6 +54,23 @@ We can accomplish this using the following command:
 The new schema-based ``bids-validator`` doesn't need to be installed 
 and will be implemented automatically when `cubids validate` is called
 
+.. dropdown:: If there is no Internet connection on compute nodes
+
+    You should run one of these commands below, after installing deno, that downloads the latest version 
```

Comment:
I would make it clearer why this is necessary. Something like "If your HPC doesn't allow internet access on its compute nodes, it won't be able to run the online version of the BIDS validator. In that scenario, you need to install the BIDS validator on your HPC and then point to the installed version in your `cubids validate` calls. To do that, ..."